.github/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -1,13 +1,26 @@
 ## MAPS
 
-/tg/station currently comes equipped with five maps.
+/tg/station currently has in rotation five maps.
```

Comment:
Why change this?

Reply:
these are the maps in rotation, i feel that comes equipped means that these 5 are the only maps we ever had

Reply:
I think that's a misreading of it, but it's not a huge deal.
You've got the syntax a bit fuzzed tho,
`/tg/station currently has in rotation five maps.` 
should prob be
`/tg/station currently has five maps in rotation.`